client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -430,27 +430,35 @@ class PluginMeta extends Component {
 	}
 
 	render() {
+		const {
+			selectedSite,
+			plugin: pluginProp,
```

Comment:
Labelling a prop a prop in the name is somewhat extraneous and not too helpful; this is the plugin object that's been fetched from the dotorg API, so maybe call it `orgPlugin`?